Enable toggling of conceal

g0xA52A2A opened the pull request:
This introduces a variable named "troll_stopper_conceal" which can be
set to 0 in order to disable the use of conceal in this plugin. By
default it is set to 1 and has no change over prior behavior.

g0xA52A2A commented:
The motivation behind this is whilst I think it's a good use of the conceal feature for this plugin I *personally* don't like conceal otherwise and by using this plugin conceal gets activated.

One question is does this warrant a Vim help file being written? I appreciate this plugin's otherwise spartan simplicity. I could alternatively drop a note into the README. But I just wanted to give a minimal diff first of all to see if this is even wanted.

bruno- commented:
Hey,
the solution with a conceal feature was added relatively recently. After using it for a while I'm also not 100% happy with it as it slightly messes up some other file types that I'm using.

I was thinking of doing one of the following:

- removing this feature altogether
- or finding some other way to provide the same functionality
- or making it an opt-in (with a clear explanation), rather than a default

So, wanna change the PR so the feature is off by default?

> One question is does this warrant a Vim help file being written?

No need.

g0xA52A2A committed:
Require opt in to conceal features

Make enabling of conceal to view invisible characters explicit. This
preserves the Vim default of having conceallevel=0.

g0xA52A2A commented:
> So, wanna change the PR so the feature is off by default?

Done.

I can't think of another method to provide the functionality and I haven't been able to find a way to ignore other conceal related things once enabled.

bruno- commented:
Thanks.
